worker/include/DepLibUring.hpp

```diff
@@ -0,0 +1,98 @@
+#ifndef MS_DEP_LIBURING_HPP
+#define MS_DEP_LIBURING_HPP
+
+#ifdef MS_LIBURING_SUPPORTED
```

Comment:
I think it makes more sense this way: If not supported don't do absolutely anything.

```suggestion
#ifdef MS_LIBURING_SUPPORTED
#ifndef MS_DEP_LIBURING_HPP
#define MS_DEP_LIBURING_HPP

```